php/values.yaml

```diff
@@ -312,10 +312,10 @@ fpm:
 
   # php.ini configuration
   # http://php.net/manual/en/errorfunc.configuration.php
-  ini:
-    displayErrors: false
-    errorReporting: E_ALL
-    exposePhp: false
+  ini: |
+    display_errors = Off
+    error_reporting = E_ALL
+    expose_php = Off
```

Comment:
@cw-niinuma 
I  can see `ini` values express php.ini file. So it would be better to put configuration files into  `templates`  key as same as `php-fpm.conf`

For example are here.

```
templates:
  php-ini: |
    display_errors = Off
```